start-tests.sh

```diff
@@ -7,106 +7,267 @@
+    log_color "red" "One or more variables are undefined for imported cluster." "(set ${PURPLE}OPTIONS_MANAGED_BASEDOMAIN, OPTIONS_MANAGED_BASEDOMAIN, and OPTIONS_MANAGED_BASEDOMAIN${NC} to execute the test with environment variables)\n"
+
+    # The mount path will be set by CICD.
+    if [[ ! -z $OPTIONS_KUBECONFIG_MOUNT_PATH ]]; then
+      echo -e "Detected kubeconfig mount path for imported cluster at: $OPTIONS_KUBECONFIG_MOUNT_PATH"
+      OPTIONS_MANAGED_KUBECONFIG=$OPTIONS_KUBECONFIG_MOUNT_PATH
+    fi
+
+    # Check to see if the kubeconfig for the managed cluster is available.
+    if [[ ! -f $OPTIONS_MANAGED_KUBECONFIG ]]; then
+      log_color "red" "The kubeconfig file for imported cluster was not located." "(set ${PURPLE}KUBECONFIG${NC} to ${YELLOW}$OPTIONS_MANAGED_KUBECONFIG${NC} and oc login to create kubeconfig file)"
+      echo -e "Skipping managed cluster test.\n"
+      export CYPRESS_SKIP_MANAGED_CLUSTER_TEST=true
+    else
+      # Exporting this variable so cypress will know to use the kubeconfig file for the imported cluster.
+      export CYPRESS_USE_MANAGED_KUBECONFIG=true
+      export CYPRESS_SKIP_MANAGED_CLUSTER_TEST=false
+
+      echo -e "Kubeconfig file detected at: $OPTIONS_MANAGED_KUBECONFIG - copying to ./kube/config/import-kubeconfig\n"
+      cp $OPTIONS_MANAGED_KUBECONFIG ./kube/config/import-kubeconfig
+      export OPTIONS_MANAGED_KUBECONFIG=./kube/config/import-kubeconfig
+
+      if [[ -z $OPTIONS_MANAGED_KUBECONTEXT || "$OPTIONS_MANAGED_KUBECONTEXT" == "null" ]]; then
+        MANAGED_CLUSTER=($(oc config get-clusters --kubeconfig=$OPTIONS_MANAGED_KUBECONFIG))
+        export OPTIONS_MANAGED_KUBECONTEXT=default/${MANAGED_CLUSTER[1]}/kube:admin
+      fi
+
+      log_color "cyan" "Switching context to log into Kube API server"
+      oc config use-context --kubeconfig=$OPTIONS_MANAGED_KUBECONFIG $OPTIONS_MANAGED_KUBECONTEXT
+      export OPTIONS_MANAGED_BASEDOMAIN=$(echo ${MANAGED_CLUSTER[1]} | cut -d'-' -f2- | cut -d':' -f1)
+      export OPTIONS_MANAGED_USER=kubeadmin
+
+      log_color "purple" "IMPORTED CLUSTER:" "$OPTIONS_MANAGED_BASEDOMAIN\n"
+    fi
+  else
+    echo -e "Environment variables detected for managed cluster. Configuring tests to execute with imported cluster exported variables.\n"
+    log_color "cyan" "Logging into the managed cluster using credentials and generating the kubeconfig..."
+
+    export OPTIONS_MANAGED_URL="https://api.$OPTIONS_MANAGED_BASEDOMAIN:6443"
+
+    export KUBECONFIG=./kube/config/import-kubeconfig
+    touch $KUBECONFIG
+    
+    oc login --server=$OPTIONS_MANAGED_URL -u $OPTIONS_MANAGED_USER -p $OPTIONS_MANAGED_PASSWORD --insecure-skip-tls-verify
+    export OPTIONS_MANAGED_KUBECONFIG=$KUBECONFIG
+    unset KUBECONFIG
+
+    if [[ -f $OPTIONS_MANAGED_KUBECONFIG ]]; then
+      echo -e "Successfully detected managed cluster kubeconfig.\n"
+    else
+      echo -e "Failed to create or locate managed cluster kubeconfig.\n"
+    fi
+  fi
```

Comment:
If the `managed` cluster environment variables are exported (i.e user, basedomain, password), we can continue the test as usual and log in with the provided credentials. Similar to the `hub` cluster, we are setting the `kubeconfig` to use the path that we have set for the `managed` cluster.